qml/main.qml

```diff
@@ -42,6 +42,7 @@ Window {
     }
 
     StackView {
+        property int mccOffsetFromUTC: -1
```

Comment:
Why do we have this here? If you need to share things between two components use a static js library.